### PR TITLE
Fix: Coverity issues. 

### PR DESCRIPTION
**Pull request by randylintw**

### CODE REVIEW

**Comment by tsaikevin** [comment]
@randylintw can you add CWE # in another 2 commits, it helps us trace back to scanning report. thanks

**Review by tsaikevin — APPROVED** [review]

File: Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c [hunk 1]
```diff
@@ -183,7 +183,7 @@ GetCpuFamily (
   VOID
   )
 {
-  EFI_CPUID_REGISTER Cpuid;
+  EFI_CPUID_REGISTER Cpuid = { 0, 0, 0, 0 };
```
Comment (tsaikevin) [inline Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c:186]: the Cpuid registers get initial value in AsmCpuid routine.
wonder if scanning can analyze assembly code.

**Comment by randylintw** [comment]
> 

Sure, will change, thanks.

File: Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c (hunk 1 above)
Comment (randylintw) [reply Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/UpdateAcpiGnvs.c:186]: Will remove this commit id.

**Pull request closed without merging**